### PR TITLE
chore(macros/CSSRef): add color guides

### PR DESCRIPTION
**Pull request by estelle**
## Summary

Adds the three color guides to the CSS guides section of the CSS sidebar

### Problem
These color guides are not listed in the CSS sidebar
Added in: https://github.com/mdn/content/pull/33071

### Solution

This adds the 3 guides

### Screenshots

Screenshot of sidebar showing color guides

## How did you test this change?

localhost:5042
i clicked on all three links and the pages came up.

### CODE REVIEW

File: kumascript/macros/CSSRef.ejs [hunk 1]
```diff
@@ -1250,6 +1253,9 @@ async function buildPropertylist(pages, title) {
           <summary><%=text['Colors']%></summary>
           <ol>
             <li><%-smartLink(`${cssURL}CSS_colors/Applying_color`, null, text['Applying_color_to_HTML_elements_using_CSS'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_colors/Color_values`, null, text['Color_values'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_colors/Relative_colors`, null, text['Relative_colors'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_colors/Using_color_wisely`, null, text['Using_color_wisely'], cssURL)%></li>
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:1258]: Can you please check if this works? 🙏  (If so, we don't need the translations above.)

```suggestion
            <li><%-smartLink(`${cssURL}CSS_colors/Color_values`)%></li>
            <li><%-smartLink(`${cssURL}CSS_colors/Relative_colors`)%></li>
            <li><%-smartLink(`${cssURL}CSS_colors/Using_color_wisely`)%></li>
```

**Review by caugner — CHANGES_REQUESTED** [review]

File: kumascript/macros/CSSRef.ejs [hunk 2]
```diff
@@ -70,9 +70,12 @@ const text = mdn.localStringMap({
...
       'Colors': 'Colors',
-        'Applying_color_to_HTML_elements_using_CSS': 'Applying color to HTML elements using CSS',
-        'Web_Accessibility_Understanding_Colors_and_Luminance': 'Web Accessibility: Understanding colors and luminance',
-        'Color_contrast': 'Web Accessibility: Color contrast',
+        'Applying_color_to_HTML_elements_using_CSS': 'Applying color to HTML elements',
+        'Color_values': 'Color values',
+        'Relative_colors': 'Using relative colors',
+        'Using_color_wisely': 'Using colors wisely',
+        'Web_Accessibility_Understanding_Colors_and_Luminance': 'Accessibility: Understanding colors and luminance',
+        'Color_contrast': 'Accessibility: Color contrast',
```
Comment (caugner) [inline kumascript/macros/CSSRef.ejs:78]:
```suggestion
        'Applying_color_to_HTML_elements_using_CSS': 'Applying color to HTML elements',
        'Web_Accessibility_Understanding_Colors_and_Luminance': 'Accessibility: Understanding colors and luminance',
        'Color_contrast': 'Accessibility: Color contrast',
```

**Review by caugner — APPROVED** [review]

**Comment by caugner** [comment]
Putting in draft while the upstream PR is not merged yet.

@estelle Please mark the PR as "ready for review" again as soon as the content changes land. Thanks!

**Comment by estelle** [comment]
@caugner https://github.com/mdn/content/pull/33071 has been merged!

**Pull request merged**